README.md

```diff
@@ -100,6 +100,17 @@ Vertical or horizontal lines are supported.
 	// https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineDashOffset
 	borderDashOffset: 5,
 
+	// Only draw a line for a specific data at the specified index. Currently only supported for bar graphs
```

Comment:
data is plural, but it sounds like this option applies only to a single element, so it might be better to say "specific data element" instead